### DISCUSSION TITLE
Github pages loading very slow

### DISCUSSION THREAD

**Discussion by nuuuclear** [community/community#23426] [Other Feature Feedback, Questions, & Ideas]
<p>I am making a portfolio type website (<a href="https://devarite.github.io" rel="noopener nofollow ugc">https://devarite.github.io</a>) on github pages but 2 days ago it randomly started loading very slowly and sometimes not at all, i’ve tried removing unused things but nothing i did stopped it. any tips to improve performance?</p>

**Comment by yoannchaudet** [comment]
<p>Hello!</p>
<p>I took a quick look and your Pages is waiting on this script to load: <a href="https://kit.fontawsome.com/c08a4915fa.js">https://kit.fontawsome.com/c08a4915fa.js</a>.</p>
<p>You may want to swap it by something else or just remove it (I assume it is a webfont).</p>
<p>I blocked the request in my browser and could see your Pages load as fast as it should:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/5/15d30183c329780626d896937d360334fbc35d28.png" data-download-href="/uploads/short-url/3746vvzotOmaODkYO27P7inLQ9O.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181076874-137df4c6-995a-45be-abe5-840d2cdb7c5c.png" alt="image" data-base62-sha1="3746vvzotOmaODkYO27P7inLQ9O" width="690" height="333" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15d30183c329780626d896937d360334fbc35d28_2_690x333.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15d30183c329780626d896937d360334fbc35d28_2_1035x499.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15d30183c329780626d896937d360334fbc35d28_2_1380x666.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/5/15d30183c329780626d896937d360334fbc35d28_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">4122×1990 552 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by nuuuclear** [comment]
<p>That worked, it was an icon kit with thousands of images</p>
